components/swap/review.vue

```diff
@@ -134,14 +135,16 @@ const submit = () => {
     duration: swap.value.duration,
     surcharge: swap.value.surcharge,
     urlPrefix: urlPrefix.value,
-    successMessage: $i18n.t('swap.created'),
   })
 }
 
-watchEffect(async () => {
+watch([status, blockNumber], () => {
   if (status.value === TransactionStatus.Finalized && blockNumber.value) {
+    successMessage($i18n.t('swap.created'), {
+      footer: computed(() => ({ icon: 'circle-info', label: $i18n.t('general.updateOnWebsiteSoon') })),
```

Comment:
would make the footer `MaybeRef`

```suggestion
      footer: { icon: 'circle-info', label: $i18n.t('general.updateOnWebsiteSoon') },
```

Reply:
updated